### ISSUE TITLE
GStreamer JACK audio plugin is in good or good-extras

### ISSUE THREAD

**Issue by fedelibre** [jackaudio/jackaudio.github.com#50]
On [gstreamer_via_jack](http://jackaudio.org/faq/gstreamer_via_jack.html) page it's stated:

>You need the GStreamer JACK audio plugin, which is currently (Fall 2009) part of the "bad" plugins collection. 

But I found out that the plugin is currently (as of version 1.8.1) in the [good plugins](https://gstreamer.freedesktop.org/data/doc/gstreamer/head/gst-plugins-good-plugins/html/gst-plugins-good-plugins-plugin-jack.html).

I would suggest searching the repository for the package providing the relevant file. On Fedora23 it's currently in good-extras:

```
# dnf provides libgstjack.so
Last metadata expiration check: 2:08:53 ago on Sun Apr 24 21:21:36 2016.
gstreamer1-plugins-good-extras-1.6.3-1.fc23.i686 : Extra GStreamer plugins with good code and licensing
Repo         : updates

gstreamer-plugins-good-extras-0.10.31-15.fc23.i686 : Extra GStreamer plug-ins with good code and
                                                   : licensing
Repo         : fedora

gstreamer1-plugins-good-extras-1.6.0-1.fc23.i686 : Extra GStreamer plugins with good code and licensing
Repo         : fedora
```

Should I send a pull request?

**Comment by fedelibre** [comment]
One more thing: gconf2 is obsolete and it's been replaced by dconf-editor and gsettings.
But I cannot find any gstreamer schema:

```
$ gsettings list-schemas | grep gstreamer
org.gnome.gthumb.gstreamer-tools
```

I'm stuck

**Comment by mseeber** [comment]
The tutorial needs an update.

For the second comment:
If this is about setting up gstreamer, please contact the maintainers/developers of gstreamer, as I am not familiar with setting up gstreamer.

**Comment by fedelibre** [comment]
I was wrong in the second comment. If I install gconf2, I can find the schema.
But yesterday something didn't work anyway. I'll try again as soon as I have some time.

**Comment by mseeber** [comment]
So basically the only point that need change is the fact that it is in "bad" plugins?

**Comment by fedelibre** [comment]
Yes, it's not in the bad plugins but in the good or good-extras (as in Fedora 23) plugins.

I realized why the tutorial didn't work for me. I don't have `jackaudiosink` in my path and I cannot find any package which contains it. I found [other people having the same problem years ago](https://lists.fedoraproject.org/pipermail/users/2011-September/404787.html). And [here](https://bugzilla.redhat.com/show_bug.cgi?id=714481) it seemed to be fixed in Fedora.

I think that I'll have to file a bug to Fedora?

Here's what I checked:

```
$ gst-inspect | grep jack
]$ gst-inspect jackaudiosink
No such element or plugin 'jackaudiosink'
```

**Comment by mseeber** [comment]
Yes, a report against the gstreamer package is probably appropriate.
